src/StructuredLogger/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -422,6 +471,7 @@ private void Write(BuildMessageEventArgs e)
         {
             switch (e)
             {
+                case FileUsedEventArgs responseFileUsed: Write(responseFileUsed); break;

```

Comment:
why did FileUsed move to the top? I'm not sure if the order here matters and how the switch statement works for types under the covers. Generally I'd put most common event args on top, and FileUsed is probably the least commonly used of them all.

Reply:
Makes sense.
Though I'd keep the same ordering as in MSBuild for less painfull diffing.
IIRC it should translate to dictionary - so hopefully no harm

Reply:
Sure, if MSBuild does that, we should do the same think. I'm also not sure if the order even matters here.